jobboard/views.py

```diff
@@ -38,13 +37,38 @@ def get_content_if_user_is_not_student(request, user_is_recruiter):
     return content
 
 
+def searched_by_companies_or_keywords(searched_name):
+    jobs_by_company = Job.get_jobs_by_company_name(searched_name)
+    jobs_by_keywords = Job.get_jobs_by_keywords(searched_name)
+    return jobs_by_company.union(jobs_by_keywords)
+
+
+def searched_by_area(searched_area):
```

Comment:
why not search by city? if that what we do here? or if that was a region so search by region?

Reply:
You're right, I'll change it